aws-test/tests/aws_rds_db_cluster/test-hydrate-expected.json

```diff
@@ -8,9 +8,9 @@
     "db_subnet_group": "{{ resourceName }}",
     "deletion_protection": false,
     "endpoint": "{{ output.endpoint.value }}",
-    "engine": "aurora",
+    "engine": "aurora-mysql",
     "engine_mode": "provisioned",
-    "engine_version": "5.6.mysql_aurora.1.22.5",
+    "engine_version": "5.7.mysql_aurora.2.11.2",
```

Comment:
Since these output parameters will keep on varying depending on the terraform and the provider version, I feel it is better to remove these parameters from the `expected.json` file.